### ISSUE TITLE
Add search to admin users page

### ISSUE THREAD

**Issue by robines** [Samfundet/Samfundet4#1494]
(reserved for nyopptatt)

Depends on #1493 

Add a text input field to the page. After inputing something, the input should be sent as a parameter to the "all users" endpoint to filter the results.

**Comment by Snorre98** [comment]
Another issue (https://github.com/Samfundet/Samfundet4/issues/1650) asks for similar functionality, but in a different usecase. The same API call might be used in the solution for both this (#1494) and #1650
